### ISSUE TITLE
Required Taskotron checks should give me a list of options

### ISSUE THREAD

**Issue by hroncok** [fedora-infra/bodhi#3647]
When I enter new update via https://bodhi.stg.fedoraproject.org/updates/new it let's me select taskotorn checks that are required. But I need to type them to a free text form. It should give me options instead.

**Comment by ryanlerch** [comment]
I had a dig into the taskotron API, but can't figure out if we can get this list to populate this control with.

does anyone know anything about taskotron?

**Comment by hroncok** [comment]
> does anyone know anything about taskotron?

@kparal does

**Comment by kparal** [comment]
No, it's not possible to get a list of checks in an automated fashion. Also, some checks only run in certain environments (e.g. not on Rawhide, etc). But the bigger problem is that Taskotron is likely going away in the near future, and we will hopefully move some of our tasks into Fedora CI. There are other likely candidates for generic tests that are going to be run by Fedora CI & comp., like [rpminspect](https://github.com/rpminspect/rpminspect). So I guess this will need some general solution, not tied to Taskotron.
